code/modules/antagonists/superheroes/buzzon/buzzon_equipment.dm

```diff
@@ -0,0 +1,294 @@
+//Buzzon, controls all the robots and honey.
+
+/obj/item/clothing/gloves/tackler/combat/insulated/buzzon
+	name = "BuzzOn gloves"
+	desc = "Legendary gloves on the greatest bee superhero - BuzzOn. You can see small rocket engines installed on them."
+	icon_state = "buzz"
+	worn_icon_state = "buzz"
+	inhand_icon_state = "ygloves"
+
+	armor = list(MELEE = 60, BULLET = 60, LASER = 50, ENERGY = 60, BOMB = 55, BIO = 100, RAD = 70, FIRE = 100, ACID = 100, WOUND = 25)
+
+	tackle_stam_cost = 5 //Very light and fast tackles. However, if you fly into a wall or an enemy, well, you're fucked.
+	base_knockdown = 0.2 SECONDS
+	tackle_range = 5
+	tackle_speed = 2
```

Comment:
does leaping around with these gloves actually get you around the station faster than just running? they knock you down for 0.2 seconds each time you use them, and you'll need to spend a second getting up each time you get knocked down...

can you tackle while prone?